docs/README.md

```diff
@@ -28,11 +28,11 @@ Run as a JAR, using Docker, or on [Metabase Cloud](https://store.metabase.com/ch
 
 Once installed, set up your Metabase and connect to your data.
 
-### [Getting started](https://www.metabase.com/learn/getting-started/getting-started)
+### [Getting started](https://www.metabase.com/learn/metabase-basics/getting-started/index)
```

Comment:
```suggestion
### [Getting started](https://www.metabase.com/learn/metabase-basics/getting-started)
```
Do we need the index?

Reply:
I just copy-pasted the resolved link from the address bar. It'll work without the index but I just want to reduce redirects as much as possible